src/rewriter.fs

```diff
@@ -964,7 +969,29 @@ let rec private iterateSimplifyAndInline (options: Options.Options) optimization
             iterateSimplifyAndInline options optimizationPass (passCount + 1) li
         else li
 
-let simplify options li =
+let simplify (options: Options.Options) li =
+    let mutable forceInlineNextFunction = None
+    let processPragmas tl =
+        match tl with
+        | TLDirective ["#pragma function inline"] ->
```

Comment:
String matching is very restricted and will fail if there's any extra whitespace.
In theory, we could parse the pragmas in the parser.

Reply:
I'll accept multiple spaces. But I'm not so sure that the parser is the best place to process a format that's specific to minifier rather than specific to the language.